### DISCUSSION TITLE
How to get current amount of seeds?

### DISCUSSION THREAD

**Discussion by AskAlexSharov** [anacrolix/torrent#738] [Q&A] [answered]
I see method: client.Torrents()[0].PeerConns()
But no method for SeedConns
Or need do another way?

**Comment by anacrolix** [comment, marked as answer]
Maybe filter the PeerConns on PeerConn.PeerPieces.GetCardinality()==Torrent.NumPieces? I'm not sure how efficient that is, PeerPieces, may allocate heavily, let me know how it goes.
